<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuclear Power Plant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="buttonReactor">
      <property name="text">
       <string>Reactor</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="8">
     <widget class="QTextEdit" name="textAlarms"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="buttonStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="buttonOfflineCooling">
      <property name="text">
       <string>Offline Cooling</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="buttonLoop2">
      <property name="text">
       <string>Reactor Loop 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="buttonRestart">
      <property name="text">
       <string>Restart</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="buttonDeareator">
      <property name="text">
       <string>Deareator</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelRunning">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="buttonCondenser">
      <property name="text">
       <string>Condenser</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonFeedWater">
      <property name="text">
       <string>Feed Water</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="buttonECC">
      <property name="text">
       <string>Emergency Cooling</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelAlarms">
      <property name="text">
       <string>Alarms</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="buttonTurbineSupport">
      <property name="text">
       <string>Turbine Support</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="buttonSystemSummary">
      <property name="text">
       <string>System Summary</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QPushButton" name="buttonAlarmsReset">
      <property name="text">
       <string>Reset alarms</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="buttonControlRods">
      <property name="text">
       <string>Control Rods</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="buttonLoop1">
      <property name="text">
       <string>Reactor Loop 1</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="buttonTurbine">
      <property name="text">
       <string>Turbine</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
